wiki/History_of_osu!/2007/en.md

```diff
@@ -28,13 +28,13 @@ By the end of July, the menu system was changed to add a bit of style to the gam
 
 ## August
 
-The main menu was updated with a new look, and the editor received a lot of new functionalities, including copy and paste support. The Windows's menubar and cursor were added, making editor features were a lot more accessible than before. Sliders were more extensively supported, with ticks present and disappearing as the slider ball passes them. Stacking of hit circles were present. Local high scores were a lot better supported. The `.osu` file format was revised and completely revamped, but includes conversion of older files to the new format.
+The main menu was updated with a new look and the editor received a lot of new functionalities including copy-and-paste support. The Windows's menubar and cursor were added, making editor features way lot more accessible than before; sliders were more extensively supported with [slider ticks](/wiki/Glossary#slider-tick) present; [stacking](/wiki/Mapping_Techniques/Stack) of [hit circles](/wiki/Hit_Objects#hit-circle) were present; local high scores were better supported; and the `.osu` file format was revised and completely revamped, but included conversion of older files to the new format.
```

Comment:
`making editor features way lot more accessible than before`

This seems like it reads weird.

Reply:
Will apply soon™